chapters/introduction.tex

```diff
@@ -72,8 +72,33 @@ \section{Some Definitions}\label{some-definitions}
 and components, and generation of connection equations from \lstinline!connect!-equations (basically, mapping the hierarchical structure of a model into a set of differential, algebraic and discrete equations together with the corresponding variable declarations and function definitions from the model).
 \end{definition}
 
+\begin{definition}[Initialization]\index{initialization}
+The simulation starts with solving the initialization problem at the starting time giving values for all variables that are consistent with the result of the flattening.
+\end{definition}
+
+\begin{definition}[Transient analysis]\index{transient analysis}\index{initial value problem!see{transient analysis}}
+Starting from the result of the initialization problem the model is then simulated forward in time.
+This generates values for the variables at different times -- and uses numerical methods for handling the hybrid DAE generated by the flattening.
+\end{definition}
```

Comment:
How about a cross reference to `\cref{modelica-dae-representation}` here?

Reply:
Yes, I'll add a cross-reference a few lines above.

Reply:
OK.  Resolving.